taichi/backends/metal/codegen_metal.cpp

```diff
@@ -270,6 +270,11 @@ class KernelCodegen : public IRVisitor {
          stmt->val->raw_name());
```

Comment:
We'll one day make another PR to remove `ArgStore` and `mark_as_return_value` completely.

Reply:
Yep, sounds good. FWIW, I don't think `ArgStoreStmt` is in use anymore, so maybe it's fine to just remove it after this one?